client/src/views/admin/addCountry.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+    <div>
+        <b-card>
+            <b-container>
+                <h1>
+                    <b>ADD COUNTRY</b>
+                </h1>
+                <div class="form mb-3">
+
+                    <b-form-input v-model="country.name" placeholder="Enter the CountryName" required></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="country.description" placeholder="Description" required></b-form-input>
+                </div>
+                <div class="form mb-3">
+                    <b-form-input v-model="country.countryCode" placeholder="Country Code" required></b-form-input>
+                </div>
+                <b-button variant="outline-primary" size="lg" @click="handleAdd">Add</b-button>
+            </b-container>
+        </b-card>
+        <table class="table">
+            <thead>
```

Comment:
`Vue js` is used to utilise maximum component usage, its better to create multiple components for create/update/delete instead of writing code for create/ update/ delete/ listing in same page, we should decrease loc always ;)